lucet-runtime/lucet-runtime-internals/src/instance.rs

```diff
@@ -825,6 +881,11 @@ impl Instance {
         );
         self.state = State::Running;
 
+        {
+            let mut kill_state = self.kill_state.thread_id.lock().unwrap();
+            *kill_state = Some(unsafe { pthread_self() });
```

Comment:
Could we rename the variable used on the left-hand side, or just do the lock and assignment in a single statement?